datadistillrSDK/pythonSDK.py

```diff
@@ -0,0 +1,139 @@
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from datadistillrSDK.project import project
+
+
+class Datadistillr:
```

Comment:
In general, it is a good practice to put classes in a file with the same name.

Reply:
I wanted to name this file datadistillr.py but that file already exists. Should I rename the class and file name to "DatadistillrAccount" or should I merge the current datadistillr.py and pythonSDK.py into one file/class named "datadistillr"?

Reply:
In my most recent push I decided to rename "pythonSDK" to DatadistillrAccount"